src/play-item.js

```diff
@@ -0,0 +1,47 @@
+import {setup} from './auto-advance.js';
+
+/**
+ * Removes all remote text tracks from a player.
+ *
+ * @param  {Player} player
+ */
+const clearTracks = (player) => {
+  let tracks = player.remoteTextTracks();
+  let i = tracks && tracks.length || 0;
+
+  // This uses a `while` loop rather than `forEach` because the
+  // `TextTrackList` object is not a true array.
```

Comment:
to be explicit, `TextTrackList` is a DOM "List" and it's live. It's probably fine to not update the comment.